lib/cylc/batch_sys_manager.py

```diff
@@ -613,6 +617,10 @@ def _jobs_poll_batch_sys(self, job_log_root, batch_sys_name, my_ctx_list):
                 except IOError as exc:
                     sys.stderr.write(str(exc) + "\n")
 
+                file_ctx = self._jobs_poll_status_files(
```

Comment:
```suggestion
                # Re-read the status file in case the job started and exited
                # between the file and batch system checks, which would be
                # interpreted as submit-failed (job exited without starting).
                # Possible if polling many jobs and/or system heavily loaded.
                file_ctx = self._jobs_poll_status_files(
```